packages/phone-number-privacy/src/common/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Response } from 'firebase-functions'
+
+export enum ErrorMessages {
+  DATABASE_UPDATE_FAILURE = 'CELO_PNP_ERR_01 Failed to update database entry',
+  DATABASE_INSERT_FAILURE = 'CELO_PNP_ERR_02 Failed to insert database entry',
+  DATABASE_GET_FAILURE = 'CELO_PNP_ERR_03 Failed to get databse entry',
+  INVALID_INPUT = 'CELO_PNP_ERR_04 Invalid input paramaters',
+  EXCEEDED_QUOTA = 'CELO_PNP_ERR_05 Requester exceeded salt service query quota',
+  SALT_COMPUTATION_FAILURE = 'CELO_PNP_ERR_06 Failed to compute BLS salt',
+  UNKNOWN_ERROR = 'CELO_PNP_ERR_07 Something went wrong',
+}
+
+export function respondWithError(res: Response, statusCode: number, error: string) {
+  res.status(statusCode).json({ success: false, error })
+}
+
+export class HashSet<T> {
```

Comment:
What's this for? Seems like a recreation of `Map` which node already has

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
i'm trying to get O(N) lookup. that Map.has() looks promising. if its underlying data structure is a javascript map, then we are probably good

Reply:
O(1)*

Reply:
yeah Map.has is O(1)